File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.py
def solution(A, B):
    if A == B:
        answer = 0
    else:
        for i in range(len(A)):
            A = A[-1] + A[:len(A)-1]
            if A == B:
                answer = i+1
                break
    if A != B:
        answer = -1
    return answer

File: 프로그래머스/0/120921. 문자열 밀기/문자열 밀기.java
class Solution {
    public int solution(String A, String B) {
        int answer = -1;
        if (A.equals(B)) {
            answer = 0;
        } else {
            int len = A.length();
            for (int i = 1; i < len; i++) {
                A = A.substring(len-1) + A.substring(0, len -1);
                if (A.equals(B)) {
                    answer = i;
                    break;
                }
            }
        }
        return answer;
    }
}